### PR TITLE
Gui integration

### PR DESCRIPTION
**Pull request by Azaken1248**
Changing Input Array Type From String To JButton

### CODE REVIEW

File: Slide/src/Slide_Game.java [hunk 1]
```diff
@@ -31,9 +32,9 @@ public class Slide_Game  implements ActionListener,MouseListener{
 	JLabel Time_Label;
 	JLabel Moves;
 	
+	JButton[] clickObj = new JButton[9];
 	
-	
-	File file = new File("R.jpeg");
+	File file = new File("C:/Users/Rohit Sinha/Desktop/Slide/slide/Slide/assets/R.jpeg");
```
Comment (poseidon0z) [inline Slide/src/Slide_Game.java:37]: don't use absolute path

File: Slide/src/Slide_Main.java [hunk 2]
```diff
@@ -13,7 +14,7 @@
 
 public class Slide_Main {
 	public static void main(String[]args) throws IOException{
-		/*File file = new File("R.jpeg");
+		/*File file = new File("C:\\Users\\Rohit Sinha\\SlideGameGUI\\SlideGameGUI\\R.jpeg"); //enter your own path here
```
Comment (poseidon0z) [inline Slide/src/Slide_Main.java:17]: Here too... no absolute path

**Review by poseidon0z — CHANGES_REQUESTED** [review]
Need 2 changes from absolute to relative paths to allow cross device debugging

File: Slide/src/Slide_Game.java (hunk 1 above)
Comment (poseidon0z) [reply Slide/src/Slide_Game.java]: Resolved

File: Slide/src/Slide_Main.java (hunk 2 above)
Comment (poseidon0z) [reply Slide/src/Slide_Main.java:17]: Resolved

**Review by poseidon0z — APPROVED** [review]
Changes approved with changes

**Review by Azaken1248 — COMMENTED** [review]
OK

**Pull request merged**